README.md

```diff
@@ -160,7 +160,7 @@ sudo emerge net-misc/curl \
     media-libs/freetype media-libs/mesa dev-util/gperf \
     dev-python/virtualenv dev-python/pip dev-libs/openssl \
     media-libs/harfbuzz dev-util/ccache sys-libs/libunwind \
-    x11-libs/libXmu media-libs/glu x11-base/xorg-server sys-devel/clang \
```

Comment:
Not 100% sure about this one, I'm not an Arch user so I'm not sure how these packages match up against the other ones.